lib/fedex/pending_shipment_label.rb

```diff
@@ -0,0 +1,11 @@
+
+module Fedex
+  class PendingShipmentLabel
+    attr_accessor :email_label_url
+    def initialize(label_details = {})
+      @email_label_url = label_details[:completed_shipment_detail][:access_detail][:email_label_url]
+    end
+
+
+  end
```

Comment:
Seems like quite much extra space, IMHO :)

I think it would be a bit more readable if we'll put a line in between function definition and attr_accessor instead. What do you think?